src/App.js

```diff
@@ -0,0 +1,23 @@
+import ReactDOM from "react-dom";
+import React from "react";
+import {Widget} from "./widget/widget";
+
+
+const e = React.createElement;
+
+export class App extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+    render(){
+        return(
+        <Widget>
```

Comment:
you can jsut do <Widget />

Reply:
What do you mean?

Reply:
sorry it seems that Github did not pick up the rest of my comment, I had typed: `<Widget />`